azure-devops/projects/selfcare-projects/selc-fe-login.tf

```diff
@@ -33,7 +33,12 @@ locals {
   }
   # deploy vars
   selc-fe-login-variables_deploy = {
-
+    dev_react_app_url_file_informativa_privacy  = "https://dev.selfcare.pagopa.it/assets/InformativaPrivacy.pdf"
+    dev_react_app_url_file_termini_e_condizioni = "https://dev.selfcare.pagopa.it/assets/tc.pdf"
```

Comment:
@antonioT90 could you change this url with extended name for tc.pdf and we could use always english name for vars name
ie dev_react_app_url_file_informativa_privacy to dev_react_app_url_file_privacy_disclaimer
ie dev_react_app_url_file_termini_e_condizioni to dev_react_app_url_file_terms_and_conditions

Reply:
renamed